bpfd-operator/pkg/helpers/helpers.go

```diff
@@ -305,6 +305,31 @@ func GetMaps(c *bpfdclientset.Clientset, ProgramName string, mapNames []string)
 // 	return nil
 // }
 
+// Returns true if loaded.  False if not.  Also returns the condition type.
+func isProgLoaded(conditions *[]metav1.Condition) (bool, string) {
+	// Get most recent condition
+	conLen := len(*conditions)
+
+	if conLen <= 0 {
+		return false, "None"
+	}
+
+	if conLen > 1 {
```

Comment:
We've decided in our core design that we should only ever have one condition. 

I don't think we should be "assuming" this may not hold true in a public library that we've designed for people to use. 

i.e this function should return the single condition `conditions[0]` and return (false, error)  or (false, "None") otherwise

Reply:
That's what it's doing -- I'm just logging a message if the condition exists.  However, I'm okay with removing the check and log here in the public library.